admin/sql/create_indexes.sql

```diff
@@ -57,4 +57,6 @@ CREATE INDEX user_id_ndx_pinned_recording ON pinned_recording (user_id);
 CREATE INDEX release_mbid_ndx_release_color ON release_color (release_mbid);
 CREATE UNIQUE INDEX caa_id_ndx_release_color ON release_color (caa_id);
 
+CREATE INDEX user_id_ndx_user_setting ON user_setting (user_id);
```

Comment:
Can use `CREATE UNIQUE INDEX` instead of declaring attribute as UNIQUE in create_tables.sql. This also matches how we do it elsewhere.